frontend/src/pages/settings/pages/destruction-report/DestructionReportSettingsPage.tsx

```diff
@@ -168,9 +169,10 @@ export function DestructionReportSettingsPage() {
       { allowClose: false },
     );
     await clearChoicesCache();
+    revalidate();
```

Comment:
Why can't the entire revalidation flow be an action, which would automatically trigger revalidation?